packages/mongo-nav/src/MongoNav.tsx

```diff
@@ -121,110 +122,70 @@ export default function MongoNav({
     projects,
   } = data;
 
-  const accountHost = hosts?.account ?? `https://account.mongodb.com`;
-  const cloudHost = hosts?.cloud ?? `https://cloud.mongodb.com`;
-  const universityHost = hosts?.university ?? `https://university.mongodb.com`;
-  const supportHost = hosts?.support ?? `https://support.mongodb.com`;
-
-  const sanitizedHosts: Required<HostsInterface> = {
-    account: accountHost,
-    cloud: cloudHost,
-    university: universityHost,
-    support: supportHost,
-    charts: hosts?.charts ?? `https://charts.mongodb.com`,
-    stitch: hosts?.stitch ?? `https://stitch.mongodb.com`,
+  const defaultHosts: Required<HostsInterface> = {
```

Comment:
Can move out of the component so we don't redefine it every time.

Reply:
👍